sdk/textanalytics/Azure.AI.TextAnalytics/src/HealthcareEntity.cs

```diff
@@ -14,15 +10,15 @@ namespace Azure.AI.TextAnalytics
     /// </summary>
     public class HealthcareEntity
     {
-        internal HealthcareEntity(HealthcareEntityInternal entity, IReadOnlyDictionary<HealthcareEntity, HealthcareEntityRelationType> relatedEntities)
+        internal HealthcareEntity(HealthcareEntityInternal entity)
         {
             Category = entity.Category;
             Text = entity.Text;
             SubCategory = entity.Subcategory;
             ConfidenceScore = entity.ConfidenceScore;
             Offset = entity.Offset;
             DataSources = entity.Links;
-            RelatedEntities = relatedEntities;
+            RelatedEntities = new Dictionary<HealthcareEntity, HealthcareEntityRelationType>(entity.RelatedEntities);
```

Comment:
`new Dictionary<HealthcareEntity, HealthcareEntityRelationType>();` will be better as later the code is responsible to setting this value.
that also means you can delete the dictionary you added to `HealthcareEntityInternal`